components/style/stylesheets.rs

```diff
@@ -103,18 +120,59 @@ pub struct NamespaceRule {
     pub url: Namespace,
 }
 
+impl ToCss for NamespaceRule {
+    // https://drafts.csswg.org/cssom/#serialize-a-css-rule CSSNamespaceRule
+    fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+        try!(dest.write_str("@namespace "));
+        if let Some(ref prefix) = self.prefix {
+            try!(dest.write_str(&*prefix));
+            try!(dest.write_str(" "));
+        }
+        try!(dest.write_str(&*self.url));
```

Comment:
We must wrap this in `url("...")`. A plain quoted string will do too, but Firefox picks the quoted URL.

https://developer.mozilla.org/en-US/docs/Web/CSS/@namespace